homeassistant/components/sensor/tof.py

```diff
@@ -0,0 +1,131 @@
+"""
+Platform for Time of Flight sensor VL53L1X.
+
+For more details about this platform, please refer to
+https://home-assistant.io/components/sensor.tof/
```

Comment:
We no longer include the url to the docs in the docstring. Just keep the very first line.

Reply:
Done